### ISSUE TITLE
vscode-insiders@1.75.0-insider: hash check failed

### ISSUE THREAD

**Issue by Elkiwa** [ScoopInstaller/Versions#840]
> scoop download vscode-insiders
> INFO  Starting download for vscode-insiders...
> Starting download with aria2 ...
> Download: Download Results:
> Download: gid   |stat|avg speed  |path/URI
> Download: ======+====+===========+=======================================================
> Download: 07d960|OK  |   9.8MiB/s|C:/Users/****/scoop/cache/vscode-insiders#1.75.0-insider#https_update.code.visualstudio.com_latest_win32-x64-archiv
> e_insider_dl.7z
> Download: Status Legend:
> Download: (OK):download completed.
> Checking hash of dl.7z ... ERROR Hash check failed!
> App:         versions/vscode-insiders
> URL:         https://update.code.visualstudio.com/latest/win32-x64-archive/insider#/dl.7z
> First bytes: 50 4B 03 04 14 00 00 00
> Expected:    817768e1651bd64fd064df0a1a9606f58e7d66432a2ecc5eb171bf4d0bd6d793
> Actual:      b972b09d135a50c1b3c73d0249c6f933ca5ca50b1cd9bd6ee0a202cd02b7cc6d
> 
> Please try again or create a new issue by using the following link and paste your console output:
> https://github.com/ScoopInstaller/Versions/issues/new?title=vscode-insiders%401.75.0-insider%3a+hash+check+failed

**Comment by ev-dev** [comment]
@Elkiwa did the bot's hash fix work for you? My vscode-insiders manifest changes were merged yesterday but it was difficult to test because VSCode doesn't seem to have any record of hashes for previous insider releases other than the latest.

**Comment by Elkiwa** [comment]
@ev-dev It did work. However, my installed extensions didn't get recognized by vscode starting from v1.75.1. Switching to insiders' version didn't solve the problem, so I intended to roll back to the previous version.

Now I am using the version that I downloaded from the official site without using scoop. Anyway, I feel like the scoop installed manifests (both insiders and normal) are probably broken on reading the contents of `C:\Users\user.name\.vscode\extensions`, though I didn't investigate further.
